python/lsst/eotest/sensor/MaskedCCD.py

```diff
@@ -181,6 +181,13 @@ def bias_image_using_overscan(self, amp, overscan=None, **kwargs):
         if overscan is None:
             overscan = self.amp_geom.serial_overscan
         try:
+            if 'bias_method' in kwargs:
+                if kwargs['bias_method'] == 'rowcol':
+                    serial_overscan   = self.amp_geom.serial_overscan
+                    parallel_overscan = self.amp_geom.parallel_overscan
```

Comment:
Using `self.amp_geom.parallel_overscan` here omits the prescan pixels so that the structure in the prescan serial direction doesn't get corrected.   I'd suggest adding a line
```
parallel_bbox = lsst.geom.Box2I(lsst.geom.Point2I(0, parallel_overscan.getMin().y), parallel_overscan.getMax())
```
then in function call below, do
```
return imutils.bias_image_rowcol(self._deep_copy(amp), amp_bbox=amp_bbox,
                                 serial_overscan=serial_overscan, parallel_overscan=parallel_bbox, **kwargs)
```

Reply:
Maybe in MaskedCCD the amps all start at `x=0` - to be safe can we accomplish that with `self.amp_geom.full_segment.getMin().x`?

Reply:
I honestly don't think that's necessary.